overlord/devicestate/handlers_install.go

```diff
@@ -72,20 +80,30 @@ func (m *DeviceManager) doSetupRunSystem(t *state.Task, _ *tomb.Tomb) error {
 		return err
 	}
 	if useEncryption {
+		fdeDir := "var/lib/snapd/device/fde"
 		args = append(args,
 			// enable data encryption
 			"--encrypt",
 			// location to store the keyfile
-			"--key-file", filepath.Join(boot.InitramfsUbuntuBootDir, "ubuntu-data.keyfile.unsealed"),
+			"--key-file", filepath.Join(boot.InitramfsEncryptionKeyDir, "ubuntu-data.keyfile.sealed"),
```

Comment:
Given that the on disk keyfile should always be sealed, is the ".sealed" necessary?

Reply:
I'd be happy with calling it just .key. @pedronis may have an opinion here.

Reply:
we have a recovery.key that is not tpm-sealed, I think it adds some clarity even it's a bit redundant